doc/source/release/1.22.0-notes.rst

```diff
@@ -3,43 +3,444 @@
 ==========================
 NumPy 1.22.0 Release Notes
 ==========================
+NumPy 1.22.0 is a big release featuring the work of 150 contributers spread
+over 574 pull requests. There have been many improvements, highlights are:
 
+* Annotations are essentially complete. Upstream is a moving target, so there
```

Comment:
```suggestion
* Annotations of the main namespace are essentially complete. Upstream is a moving target, so there
```
Might be good to explicitly mention "main namespace" or something of the sort, 
as a number subpackages (`np.ma`, `np.polynomial`, _etc._) haven't been touched at all.